packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -45,7 +45,25 @@ const _allInstances: {
   [key: string]: FocusZone;
 } = {};
 
-const _outerZones: Set<FocusZone> = new Set();
+const outZones = {
```

Comment:
Suggestion: rename to `outerZones` so that we don't have outzones and outerzones.

Reply:
Done